.travis.yml

```diff
@@ -6,8 +6,7 @@ jdk:
 
 # Run Docker Pulls in parallel, esp for vitess/base with size > 1GB
```

Comment:
Not too familiar with the travis structure but is this comment still relevant after this change?